src/test/groovy/de/tracetronic/jenkins/plugins/ecutestexecution/steps/TestRestApiClient.groovy

```diff
@@ -0,0 +1,44 @@
+package de.tracetronic.jenkins.plugins.ecutestexecution.steps
+
+import de.tracetronic.jenkins.plugins.ecutestexecution.clients.RestApiClient
+import de.tracetronic.jenkins.plugins.ecutestexecution.clients.model.ApiException
+import de.tracetronic.jenkins.plugins.ecutestexecution.clients.model.ExecutionOrder
+import de.tracetronic.jenkins.plugins.ecutestexecution.clients.model.ReportGenerationOrder
+import de.tracetronic.jenkins.plugins.ecutestexecution.clients.model.ReportInfo
+import de.tracetronic.jenkins.plugins.ecutestexecution.clients.model.TGUploadOrder
+import de.tracetronic.jenkins.plugins.ecutestexecution.model.CheckPackageResult
+import de.tracetronic.jenkins.plugins.ecutestexecution.model.GenerationResult
+import de.tracetronic.jenkins.plugins.ecutestexecution.model.UploadResult
+
+class TestRestApiClient implements RestApiClient {
```

Comment:
what are these tests for?

Reply:
Nothing -.- 
I deleted the file, because the getRestApiClient is tested by ETV1ContainerTest and ETV2ContainerTest